src/main/resources/driven-adapter/mongo-reactive/config/mongo-config.java.mustache

```diff
@@ -17,9 +17,9 @@ public class MongoConfig {
 
     {{#include-secret}}
     @Bean
-    public MongoDBSecret dbSecret(@Value("${aws.secretName}") String secret, GenericManager manager)
+    public MongoDBSecret dbSecret(@Value("${aws.secretName}") String secret, GenericManagerAsync manager)
             throws SecretException {
-        return manager.getSecretModel(secret, MongoDBSecret.class);
+        return manager.getSecret(secret, MongoDBSecret.class).block();
```

Comment:
call block function is required?

Reply:
> call block function is required?
The Client mongo has to be changed first... Issue  #128

Reply:
ok